src/xmrEngine.js

```diff
@@ -90,7 +90,7 @@ export class MoneroEngine {
     this.walletLocalDataDirty = false
     this.transactionsChangedArray = []
     this.walletInfo = walletInfo
-    this.walletId = walletInfo.id ? `${walletInfo.id} - ` : ''
```

Comment:
Has `walletId` always been suffixed with a `-` before? How has this not created bugs? Isn't this data written to disk?

Reply:
Yes, this has been a long-standing bug. However, the edge-core-js shim code has been saving us this whole time, by stomping this with the correct ID on the way out the door.